### PR TITLE
[OSDOCS-4242]: machine sets are for compute only

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.12+

Issue:
[OSDOCS-4242](https://issues.redhat.com//browse/OSDOCS-4242)

Link to docs preview:

**Architecture** > Control plane architecture > Machine roles in OpenShift Container Platform
 - [Cluster workers](https://51099--docspreview.netlify.app/openshift-enterprise/latest/architecture/control-plane.html#defining-workers_control-plane)
 - [Cluster masters](https://51099--docspreview.netlify.app/openshift-enterprise/latest/architecture/control-plane.html#defining-masters_control-plane)
 
**Installing**
 - Installing on AWS
   - Installing a cluster on AWS with customizations > Creating the installation configuration file > [Sample customized install-config.yaml file for AWS](https://51099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations.html#installation-aws-config-yaml_installing-aws-customizations) (callout 7)
   - Installing a cluster on user-provisioned infrastructure in AWS by using CloudFormation templates > Required AWS infrastructure components > [Cluster machines](https://51099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-user-infra.html#installation-aws-user-infra-cluster-machines_installing-aws-user-infra)
 - Installing on GCP
   - Installing a cluster on GCP with customizations > Creating the installation configuration file > [Sample customized install-config.yaml file for GCP](https://51099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-customizations.html#installation-gcp-config-yaml_installing-gcp-customizations) (callout 5)
 - Deploying installer-provisioned clusters on bare metal
   - Expanding the cluster > [Provisioning the bare metal node](https://51099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.html#provisioning-the-bare-metal-node_ipi-install-expanding)
 - Installing on VMC > Installing a cluster on VMC with user-provisioned infrastructure
   - [Creating the Kubernetes manifest and Ignition config files](https://51099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-user-infra.html#installation-user-infra-generate-k8s-manifest-ignition_installing-vmc-user-infra)
   - [Installing RHCOS and starting the OpenShift Container Platform bootstrap process](https://51099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-user-infra.html#installation-vsphere-machines_installing-vmc-user-infra)
 
**Post-installation configuration**
 - [Configuring a heterogeneous cluster](https://51099--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/deploy-heterogeneous-configuration.html)
 - Cluster tasks
   - [Adjust worker nodes](https://51099--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/cluster-tasks.html#post-install-adjust-worker-nodes)
   - [Creating infrastructure machine sets for production environments](https://51099--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/cluster-tasks.html#post-install-creating-infrastructure-machinesets-production)
 - Node tasks
   - Adding RHCOS compute machines to an OpenShift Container Platform cluster > [Adding a new RHCOS worker node with a custom /var partition in AWS](https://51099--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/node-tasks.html#machine-node-custom-partition_post-install-node-tasks)
 - Deploying machine health checks
   - [Understanding the difference between compute machine sets and the machine config pool](https://51099--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/node-tasks.html#differences-between-machinesets-and-machineconfigpool_post-install-node-tasks)

**Support** > Troubleshooting > Troubleshooting Windows container workload issues
 - [Investigating why Windows Machine does not become compute node](https://51099--docspreview.netlify.app/openshift-enterprise/latest/support/troubleshooting/troubleshooting-windows-container-workload-issues.html#investigating-why-windows-machine-compute-node_troubleshooting-windows-container-workload-issues)
 - Accessing a Windows node
   - [Accessing a Windows node using SSH](https://51099--docspreview.netlify.app/openshift-enterprise/latest/support/troubleshooting/troubleshooting-windows-container-workload-issues.html#accessing-windows-node-using-ssh_troubleshooting-windows-container-workload-issues)
   - [Accessing a Windows node using RDP](https://51099--docspreview.netlify.app/openshift-enterprise/latest/support/troubleshooting/troubleshooting-windows-container-workload-issues.html#accessing-windows-node-using-rdp_troubleshooting-windows-container-workload-issues)
 - [Collecting Kubernetes node logs for Windows containers](https://51099--docspreview.netlify.app/openshift-enterprise/latest/support/troubleshooting/troubleshooting-windows-container-workload-issues.html#collecting-kube-node-logs-windows_troubleshooting-windows-container-workload-issues)
 - [Collecting Windows application event logs](https://51099--docspreview.netlify.app/openshift-enterprise/latest/support/troubleshooting/troubleshooting-windows-container-workload-issues.html#collecting-windows-application-event-logs_troubleshooting-windows-container-workload-issues)
 - [Collecting Docker logs for Windows containers](https://51099--docspreview.netlify.app/openshift-enterprise/latest/support/troubleshooting/troubleshooting-windows-container-workload-issues.html#collecting-docker-logs-windows_troubleshooting-windows-container-workload-issues)

**Storage**
 - Configuring persistent storage > Persistent storage using Azure Disk > [Machine sets that deploy machines with ultra disks using PVCs](https://51099--docspreview.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent-storage-azure.html#machineset-azure-ultra-disk_persistent-storage-azure)
 - Using Container Storage Interface (CSI) > Azure Disk CSI Driver Operator > [Machine sets that deploy machines with ultra disks using PVCs](https://51099--docspreview.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-azure.html#machineset-azure-ultra-disk_persistent-storage-csi-azure)

**Operators** > Administrator tasks > Adding Operators to a cluster
 - [Pod placement of Operator workloads](https://51099--docspreview.netlify.app/openshift-enterprise/latest/operators/admin/olm-adding-operators-to-cluster.html#olm-pod-placement_olm-adding-operators-to-a-cluster)

**Machine management**
 - Overview of machine management
   - [What you can do with compute machine sets](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/index.html#what-you-can-do-with-compute-machine-sets)
   - [Autoscaler](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/index.html#autoscaler)
 - Creating compute machine sets
   - [Creating a compute machine set on Alibaba Cloud](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-alibaba.html)
   - [Creating a compute machine set on AWS](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-aws.html)
   - [Creating a compute machine set on Azure](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-azure.html)
   - [Creating a compute machine set on Azure Stack Hub](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-azure-stack-hub.html)
   - [Creating a compute machine set on GCP](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-gcp.html)
   - [Creating a compute machine set on IBM Cloud](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-ibm-cloud.html)
   - [Creating a compute machine set on Nutanix](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-nutanix.html)
   - [Creating a compute machine set on OpenStack](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-osp.html)
   - [Creating a compute machine set on RHV](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-rhv.html)
   - [Creating a compute machine set on vSphere](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-vsphere.html) 
 - [Manually scaling a compute machine set](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/manually-scaling-machineset.html)
 - [Modifying a compute machine set](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/modifying-machineset.html)
 - [Deleting a machine](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/deleting-machine.html)
 - Applying autoscaling to an OpenShift Container Platform cluster
   - [About the machine autoscaler](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/applying-autoscaling.html#machine-autoscaler-about_applying-autoscaling)
   - [Configuring the machine autoscalers](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/applying-autoscaling.html#configuring-machineautoscaler)
 - Creating infrastructure machine sets
   - [Creating infrastructure machine sets for production environments](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating-infrastructure-machinesets.html#creating-infrastructure-machinesets-production)
   - Moving resources to infrastructure machine sets
     - [Moving the router](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating-infrastructure-machinesets.html#infrastructure-moving-router_creating-infrastructure-machinesets)
     - [Moving the default registry](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating-infrastructure-machinesets.html#infrastructure-moving-registry_creating-infrastructure-machinesets)
 - [Managing machines with the Cluster API](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/capi-machine-management.html)
 - Deploying machine health checks
   - [Short-circuiting machine health check remediation](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/deploying-machine-health-checks.html#machine-health-checks-short-circuiting_deploying-machine-health-checks)
   - [Creating a MachineHealthCheck resource for bare metal](https://51099--docspreview.netlify.app/openshift-enterprise/latest/machine_management/deploying-machine-health-checks.html#mgmt-creating-mhc-baremetal_deploying-machine-health-checks) (callout 4)
 
**Nodes**
 - [Overview of nodes](https://51099--docspreview.netlify.app/openshift-enterprise/latest/nodes/index.html)
 - Controlling pod placement using node taints
   - [Adding taints and tolerations](https://51099--docspreview.netlify.app/openshift-enterprise/latest/nodes/scheduling/nodes-scheduler-taints-tolerations.html#nodes-scheduler-taints-tolerations-adding_nodes-scheduler-taints-tolerations)
 - Controlling pod placement onto nodes
   - [Placing pods on specific nodes using node selectors](https://51099--docspreview.netlify.app/openshift-enterprise/latest/nodes/scheduling/nodes-scheduler-node-selectors.html)
 - Working with nodes
   - [Understanding how to update labels on nodes](https://51099--docspreview.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-nodes-working.html#nodes-nodes-working-updating_nodes-nodes-working)
   - [Deleting nodes](https://51099--docspreview.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-nodes-working.html#deleting-nodes)

**Windows Container support for OpenShift**
 - Understanding Windows container workloads > [Windows workload management](https://51099--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/understanding-windows-container-workloads.html#windows-workload-management_understanding-windows-container-workloads)
 - Creating Windows MachineSet objects
   - [Creating a Windows MachineSet object on AWS](https://51099--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/creating_windows_machinesets/creating-windows-machineset-aws.html)
   - [Creating a Windows MachineSet object on Azure](https://51099--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/creating_windows_machinesets/creating-windows-machineset-azure.html)
   - [Creating a Windows MachineSet object on vSphere](https://51099--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.html)
 - [Windows node upgrades](https://51099--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/windows-node-upgrades.html)
 - [Scheduling Windows container workloads](https://51099--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/scheduling-windows-workloads.html)
 - Using Bring-Your-Own-Host (BYOH) Windows instances as nodes > [Configuring a BYOH Windows instance](https://51099--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/byoh-windows-instance.html#configuring-byoh-windows-instance)

**Scalability and performance**
 - [Recommended cluster scaling practices](https://51099--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/recommended-cluster-scaling-practices.html)
 - [Managing bare metal hosts](https://51099--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/managing-bare-metal-hosts.html)

**Serverless**
 - Install > Installing the OpenShift Serverless Operator > [Scaling your cluster using compute machine sets](https://51099--docspreview.netlify.app/openshift-enterprise/latest/serverless/install/install-serverless-operator.html#install-serverless-operator-scaling-with-machinesets)

**Note**
Some modules I was not able to locate in the preview:
 - modules/installation-osp-dpdk-binding-vfio-pci.adoc
 - modules/installation-osp-setting-worker-affinity.adoc
 - nodes/nodes-machinepools-about.adoc
 - osd_cluster_admin/osd_nodes/osd-nodes-machinepools-about.adoc
 - rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-about.adoc

QE review:
- [x] QE has approved this change.

Additional information:
[Background](https://docs.google.com/document/d/18gQS1sJiuL7LYEqddR8f3CTSNeG-APhrr6srNO5f_jQ/)

### CODE REVIEW

**Comment by mburke5678** [comment]
@jeana-redhat 
What you have for Nodes and WINC looks good. Are you going to update the _Creating a Windows MachineSet object..._  assemblies?  If so, I have a new assembly that I will update accordingly.

**Comment by jeana-redhat** [comment]
Thanks @mburke5678! Yes, I have updated the three existing Win machine set topics (here's [AWS](https://51099--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/creating_windows_machinesets/creating-windows-machineset-aws.html)). So if any new additions can follow that pattern that would be perfect :slightly_smiling_face:

**Comment by adellape** [comment]
LGTM for _Operators_ guide. 🚀

**Comment by sheriff-rh** [comment]
Performance/Scalability LGTM! :rocket: :sauropod: :clinking_glasses:

**Comment by Amrita42** [comment]
@jeana-redhat thanks for taking this up :) For the https://51099--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/index.html (WINC overview page), I can see 3 instances of _machine set_. I think it should be _compute machine set._

**Comment by mjpytlak** [comment]
@jeana-redhat Installing LGTM. Thanks!

**Comment by lpettyjo** [comment]
LGTM for storage.

**Review by JoelSpeed — COMMENTED** [review]
LGTM, couple of notes but I don't think they need to be resolved before this merges

File: modules/machineset-migrating-control-plane-nodes-to-diff-sd-rhv.adoc [hunk 1]
```diff
@@ -5,6 +5,7 @@
 [id="machineset-migrating-control-plane-nodes-to-diff-sd-rhv_{context}"]
 = Migrating control plane nodes to a different storage domain on {rh-virtualization}
 
+//todo: confirm if this statement is still accurate w/CPMS
```
[the remark this answers is not in the document]
Comment (jeana-redhat) [reply modules/machineset-migrating-control-plane-nodes-to-diff-sd-rhv.adoc:8]: Thanks! Will track for CPMS docs

File: modules/architecture-machine-roles.adoc [hunk 2]
```diff
@@ -47,17 +47,17 @@ In a Kubernetes cluster, the worker nodes are where the actual workloads request
...
+In {product-title}, compute machine sets control the worker machines. Machines with the worker role drive compute workloads that are governed by a specific machine pool that autoscales them. Because {product-title} has the capacity to support multiple machine types, the worker machines are classed as _compute_ machines. In this release, the terms _worker machine_ and _compute machine_ are used interchangeably because the only default type of compute machine is the worker machine. In future versions of {product-title}, different types of compute machines, such as infrastructure machines, might be used by default.
```
[the remark this answers is not in the document]
Comment (jeana-redhat) [reply modules/architecture-machine-roles.adoc:50]: Yes, I like that reconciliation - thanks!

**Comment by jeana-redhat** [comment]
@sunzhaohua2 @miyadav @huali9 This is ready for QA review. I have tagged all three of you in case you want to split up the work, but I leave it up to you to decide what approach is least onerous :slightly_smiling_face:

**Comment by sunzhaohua2** [comment]
LGTM, thanks Jeana!

**Comment by huali9** [comment]
Machine management LGTM

**Review by opayne1 — COMMENTED** [review]
Great job on this one! Just a few small things I noticed going through! :)

**Comment by jeana-redhat** [comment]
Thanks @opayne1! Those are dome good typo finds lol

**Pull request merged**